numpyro/contrib/autoguide/__init__.py

```diff
@@ -99,23 +99,27 @@ class AutoContinuous(AutoGuide):
     Base class for implementations of continuous-valued Automatic
     Differentiation Variational Inference [1].
 
-    Each derived class implements its own :meth:`get_posterior` method.
+    Each derived class implements its own :meth:`_get_transform` method.
 
     Assumes model structure and latent dimension are fixed, and all latent
     variables are continuous.
 
-    Reference:
+    .. note::
+        We recommend to use :class:`AutoContinuousELBO` as the
+        objective function `loss` in :class:`~numpyro.infer.svi.SVI`.
+        In addition, to get posterior samples from this autoguide, it
+        is better to use the :meth:`sample_posterior` method.
```

Comment:
nit: consider adding the reason why we recommend doing that. 

"We recommend using `:meth:`sample_posterior method for drawing posterior samples from the autoguide as it will automatically do any unpacking and transformations required to constrain the samples to the domain of the latent sites."

Reply:
done